core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/SnowOwl.java

```diff
@@ -73,6 +73,9 @@ public final class SnowOwl {
 	private static final String DEFAULT_DATA_PATH = "resources"; //$NON-NLS-1$
 	private static final String LOG_CONFIG_FILE = "serviceability.xml"; //$NON-NLS-1$
 	
+	// System property for additional packages to scan
```

Comment:
I would indicate in the property name that this is for component scanning. How about `-Dso.component.scan` or something similar?

Reply:
Please update the PR description section as well.